### ISSUE TITLE
Objective C API: deploymentKey argument for CodePush class

### ISSUE THREAD

**Issue by gnestor** [Microsoft/react-native-code-push#163]
Use case:

```objective-c
#ifdef DEBUG
    CodePush.deploymentKey = @"1234567890"
#else
    CodePush.deploymentKey = @"0987654321"
    jsCodeLocation = [CodePush bundleURL];
#endif
```

This automatically sets the Staging deployment key when using the Debug scheme and Production deployment key when using Release scheme. The doesn't require that developers swap out deployment keys in the Info.plist when developing and when releasing to the App Store.

It appears that the Android Java API already exposes this: `CodePush(String deploymentKey, Activity mainActivity)`

**Comment by lostintangent** [comment]
Hey @gnestor! You can actually do this today via `[CodePushConfig current].deploymentKey = ""`. I agree that it makes more sense to expose this via the `CodePush` class, and I can't send out a PR to make that change. For now, though, you can use the mentioned code snippet.

**Comment by gnestor** [comment]
It works! Great. Thanks @lostintangent and feel free to close.

**Comment by lostintangent** [comment]
Yay! I'll keep it open to track exposing this property via the `CodePush` class. I think this a cleaner and more discoverable solution. Thanks for sending this feedback! :)

**Comment by lostintangent** [comment]
Resolved with #165.
